src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocStyleCheck.java

```diff
@@ -172,6 +214,54 @@
  *   &lt;property name="checkFirstSentence" value="false"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class Test {
+ *     &#47;**
+ *      * Some description here // OK
+ *      * &lt;p // violation
```

Comment:
This is an example for unclosed tag, please change to some other sentence

Reply:
So you want me to replace  `<p // violation` with some other sentence without proper punctuation?  Correct ?

Reply:
Done